core.lua

```diff
@@ -89,14 +89,51 @@ function Simulationcraft:UpdateMinimapButton()
   end
 end
 
+local function splitLinks(links)
```

Comment:
Simpler version using regexes that might be a bit more robust:

```
local function splitLinks(links)
  local separatedLinks = {}
  for link in links:gmatch("(|c.-|h|r)") do
    separatedLinks[#separatedLinks + 1] = link
  end
  return separatedLinks
end
```

Looks for the `|c STUFF |h|r` patterns and adds them to the table.

Reply:
Done!